src/views/Transactions.vue

```diff
@@ -92,16 +111,11 @@ export default {
       if (this.$store.state.IDBReady) this.getNewTransactions()
     }
   },
-  beforeUnmount() {
-    window.removeEventListener('scroll', this.onScroll)
-  },
   mounted() {
     if (!this.$store.getters['options/isLoginViaPassword'] || this.$store.state.IDBReady) {
       this.getNewTransactions()
     }
-    window.addEventListener('scroll', this.onScroll)
   },
-  // mixins: [scrollPosition],
```

Comment:
When you navigate back from the transaction details, the scroll position must be maintained. Without this mixin, it doesn't work:

1. Browser back button: scroll position is not maintained (resets to position 0)
2. Bug: Clicking the in-app back icon redirect to the ADAMANT logo instead of the transactions list